### DISCUSSION TITLE
マイページの「質問」タブの一覧上で「クローズ済み」を表示する

### DISCUSSION THREAD

**Discussion by nak435** [increments/qiita-discussions#487] [要望・アイデア・不具合]
## 概要

マイページの「質問」タブの一覧上で「クローズ済み」を表示する。
「投稿した質問」と「回答した質問」の両方が対象。

## 期待される効果

Q&Aする方々の多くは、実質的に完了しているにも関わらず、オープンのまま放置される方が非常に多く居られます。
実際に「クローズ」されているのかが、その質問のページ移動しないと区別が無いことも、その一因と考えられるため、マイページの質問一覧で「クローズ済み」を表示すると気付きになると考えます。

さらに要望としては、「クローズ済みの非表示オプション（チェックボックス）」があれば、この上ない。

![qiita](https://github.com/increments/qiita-discussions/assets/42625437/2848b543-dedd-43b5-9c3d-30513c382414)

ぜひ実装していただきたい機能となります。よろしくお願いいたします。

**Comment by tanaka-asami415** [comment]
@nak435 
ご意見ありがとうございます🙏

> マイページの「質問」タブの一覧上で「クローズ済み」を表示する。

確かにおっしゃる通りですので、すぐの変更は難しいのですが対応したいと思います！

> 「クローズ済みの非表示オプション（チェックボックス）」があれば、この上ない。

同タイミングでの実装とならない可能性が高いですが、こちらも合わせて検討します。

**Reply by nak435** [reply]
この要望を実装していただけことに、先ほど気付いたのですが、
新機能についてのアナウンスページはどこかにあるのでしょうか？
実現していただいてありがとうございます。

**Reply by tanaka-asami415** [reply]
@nak435 
ご報告が漏れており申し訳ございません🙇

> マイページの「質問」タブの一覧上で「クローズ済み」を表示する。

こちらに関しては、下記リリースのタイミングで、ユーザーページ（マイページ）で「クローズ済み」が表示されるようになりました！
https://qiita.com/release-notes/2023-08-10

併せて、タグページの質問タブの一覧にも表示されるようになっております✨

**Reply by nak435** [reply]
ありがとうございました。
一点、変更前のような赤背景の方が目立ってよかった気もします。
普段はダークモードで見ていて、昨日までに気付かなかったのは、地味なせいかと・・・
ご検討ください。

**Reply by tanaka-asami415** [reply]
@nak435 
ご意見ありがとうございます。検討いたします🙇
